src/Apps/CollectorProfile/Routes/Saves2/Components/SavesArtworkSaveButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { SaveButtonBase } from "Components/Artwork/SaveButton/SaveButton"
+import { useState } from "react"
+
+interface SavesArtworkSaveButtonProps {
+  artworkId: string
+  collectionId: string
+}
+
+export const SavesArtworkSaveButton: React.FC<SavesArtworkSaveButtonProps> = ({
+  artworkId,
+  collectionId,
+}) => {
+  const [isSaved, setIsSaved] = useState(true)
+
+  const handleClick = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    event.preventDefault()
+    setIsSaved(!isSaved)
+
+    // TODO: Display modal window with collections
+    console.log("[debug]", artworkId, collectionId)
```

Comment:
👀

Reply:
Looks like i merged main incorrect, this is not my change
Merged again and the pr looks cleaner now